data/src/main/java/com/ericsson/bss/cassandra/ecchronos/data/sync/EccNodesSync.java

```diff
@@ -246,7 +255,30 @@ public Builder withInitialNodesList(final List<Node> nodes)
             this.initialNodesList = nodes;
             return this;
         }
-
+        /**
+         * Builds EccNodesSync with Connection Delay.
+         *
+         * @param connectionDelayValue
+         *          delay before connecting, in the unit specified in withConnectionDelayUnit
+         * @return Builder
+         */
```

Comment:
A line break in between each java doc and method

Reply:
Done